code/game/objects/items/devices/radio/radio.dm

```diff
@@ -39,6 +39,7 @@
 	var/syndie = FALSE  // If true, hears all well-known channels automatically, and can say/hear on the Syndicate channel.
 	var/list/channels = list()  // Map from name (see communications.dm) to on/off. First entry is current department (:h).
 	var/list/secure_radio_connections
+	var/list/radio_sounds = list('sephora/sound/effects/radio1.ogg','sephora/sound/effects/radio2.ogg') //Sephora - Radios make small static noises now
```

Comment:
The paths should just be sound/... rather than sephora/sound...

Reply:
Fixed